OpenFeature.sln

```diff
@@ -1,6 +1,6 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "OpenFeature.Api", "src\OpenFeature.Api\OpenFeature.Api.csproj", "{07A6E6BD-FB7E-4B3B-9CBE-65AE9D0EB223}"
```

Comment:
I think this is the right choice.

In the future, we may actually pull apart the application-author facing parts of the SDK into an "API" package, and have the full SDK consume that to keep things very small for the libraries, but there's no need for that yet.